Cargo.toml

```
@@ -40,22 +40,29 @@ plist = "0.5.3"
 prettytable-rs = "0.8.0"
 proguard = { version = "4.0.1", features = ["uuid"] }
 r2d2 = "0.8.8"
-rayon = "1.3.0"
-regex = "1.3.5"
+rayon = "1.3.1"
+regex = "1.3.9"
 runas = "0.2.1"
-rust-ini = "0.15.2"
+rust-ini = "0.15.3"
 sentry = { version = "0.18.0", default-features = false, features = ["with_client_implementation", "with_curl_transport", "with_panic", "with_failure", "with_log", "with_device_info", "with_rust_info", "with_debug_to_log"] }
-serde = { version = "1.0.105", features = ["derive"] }
-serde_json = "1.0.48"
+serde = { version = "1.0.114", features = ["derive"] }
+serde_json = "1.0.56"
 sha1 = { version = "0.6.0", features = ["serde"] }
 sourcemap = { version = "5.0.0", features = ["ram_bundle"] }
-symbolic = { version = "7.3.1", features = ["debuginfo-serde"] }
+symbolic = { version = "7.4.0", features = ["debuginfo-serde"] }
 url = "2.1.1"
 username = "0.2.0"
 uuid = { version = "0.8.1", features = ["v4", "serde"] }
 walkdir = "2.3.1"
 which = "3.1.1"
-zip = "0.5.5"
+zip = "0.5.6"
+
+[dev-dependencies]
+assert_cmd = "1.0.1"
+insta = {version= "0.16.0", features= ["redactions"]}
```

```suggestion
insta = { version = "0.16.0", features = ["redactions"] }
```

I wish rustfmt would also fmt the `toml` files